SpacetimeDB/crates/bench/hyper_cmp.py

```
@@ -0,0 +1,174 @@
+        self.rows = rows
+        self.larger = max(size) + 2
+
+
+class Stat:
+    def __init__(self, results):
+        self.spacetime = clean(results[0])
+        self.sqlite = clean(results[1])
+
+
+def load_file(named: str):
+    data = open(named).read()
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)